### PR TITLE
Update readme

### LINKED ISSUE #12: Add a Readme
The Project as no proper readme on how to use it so I have added a readme for it.

### PR DESCRIPTION
**Pull request by RitikJaiswal75**
Add basic readme

closes #12

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -13,11 +13,7 @@ export interface Env {
 }
 
 export default {
-  async fetch(
-    request: Request,
-    env: Env,
-    ctx: ExecutionContext
-  ): Promise<Response> {
```
Comment (RitikJaiswal75) [inline src/index.ts:20]: Removed these to remove lint warnings

**Review by heyrandhir — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,3 +1,45 @@
-# website-template
+# Discord Slash Command
 
-A template to create all public facing sites
+This is a discord bot for RealDevSquad discord server capable of using slash commands.
+
+This bot is written ibn TypeScript and will be hosted on cloudflare workers. Cloudflare workers provide us with a serverless environment to run our code.
```
Comment (rohan09-raj) [inline README.md:5]: Typo, also the sentence can be improved here.
```
The bot is written in
```

**Review by rohan09-raj — CHANGES_REQUESTED** [review]
We can

**Review by heyrandhir — APPROVED** [review]
LGTM

**Review by rohan09-raj — APPROVED** [review]

**Pull request merged**